### ISSUE TITLE
Improve supportive images of GUI in documentation

### ISSUE THREAD

**Issue by ninavizz** [QubesOS/qubes-issues#6471]
**The problem you're addressing (if any)**
In #6135 I noticed that some of the illustrations in the documentation to break-down and clearly identify UI components unique to Qubes, are a bit unclear. As such, I thought I'd share in here some that I made last year, for a turorial. cc'ing @deeplow should you have interest, too. 

![image](https://user-images.githubusercontent.com/8262612/111850442-91c1c300-88cd-11eb-9a77-5474fe0c35b9.png)

**Describe the solution you'd like**
Where illustrations exist in the docs that don't 100% clearly point-to and identify a named component, with a clear and clean visual hierarchy—replace one with one of these, or communicate a request to make an additional one on this ticket?

**Where is the value to a user, and who might that user be?**
Clearer images to break-down a new GUI to folks in documentation are always helpful. :)

**Describe alternatives you've considered**
Crying.

**Additional context**
Made these last year for a QubesOS training for non-technical journalist users, and am happy to make new/additional ones as documentation needs arise.

[Qubes101-2021.zip](https://github.com/QubesOS/qubes-issues/files/6174432/Qubes101-2021.zip)

**Comment by andrewdavidwong** [comment]
Very nice!

@hackerncoder, would you be willing to submit a PR that uses these images where appropriate? I ask because you're probably the most familiar with the current locations of Qubes screenshots right now. :slightly_smiling_face:

**Comment by hackerncoder** [comment]
I have become the master of the images... Yes, I'll take a look.

**Comment by ninavizz** [comment]
Thx fellas! As noted in the OG ticket, happy to create others if there's a desire. Super ez.

**Comment by ninavizz** [comment]
Thx fellas!

@hackerncoder I just looked briefly at two of the PNGs, and realized that with the updated UI the components weren't clearly enough visible through my highlights. Fixed that, in the attached PNGs here. Apologies for bein' a yo-yo, hopefully I have a credit in your bank for the laugh. 

![TrayRegions2](https://user-images.githubusercontent.com/8262612/111853172-0fd69780-88d7-11eb-80e1-5726517ba5ab.png)
![ToolbarRegionsDefined](https://user-images.githubusercontent.com/8262612/111853177-18c76900-88d7-11eb-8ac4-b57eddbbcacf.png)

**Comment by hackerncoder** [comment]
Qubes normally only comes with one Whonix, so there is only one Whonix Time sync in the tray.

**Comment by hackerncoder** [comment]
We don't show anything like CDD, NM, QU, or Whonix (from your zip file), however they are very nice and I think we should add them.

**Comment by ninavizz** [comment]
LOL @hackerncoder I have no idea what CDD, NM, or QU are. Sorry, I got involved in Qubes via another project I work on (SecureDrop) that is built on top of Qubes. Happy to provide updated screenies with only one SysWhonix, and with the other stuff omitted. 

Is the other stuff in the App Menu slide?

**Comment by hackerncoder** [comment]
Well, I just took the names in your zip file ClipboardDomainsDevices.png and shortened them.

**Comment by ninavizz** [comment]
@hackerncoder slide = corporate American jargon for "images." 

Ok, so there's nothing to explain the Tray tooling in the documentation—it's not UI elements in my customized setup of Qubes (other than the dual whonix qubes) that's at issue?

**Comment by hackerncoder** [comment]
Nope, there isn't anything to explain the tray items other than the one you saw in #6135. Not for tray and toolbar at least.

**Comment by deeplow** [comment]
Seems like I'm late to the party. Thanks @ninavizz for these goodies. Seems like @hackerncoder (the master of images) will be taking the job. 

I also feel the need for moar pictures on the docs. Made one [the other day](https://github.com/QubesOS/qubes-doc/pull/1119#issuecomment-799520147):

![111179863-5f9a2380-85a4-11eb-8b18-d3aed499354f](https://user-images.githubusercontent.com/47065258/111875461-cb5c0200-8991-11eb-9d90-ba55680f2e0e.png)

But it's not as neat as @ninavizz's.

If as part of the integrated tutorial I'm working on (https://github.com/QubesOS/qubes-issues/issues/1395) I produce some interesting images, I'll share as well to the docs.

**Comment by ninavizz** [comment]
@deeplow I studied graphic design, I can't help it, lol! Not at all late to the party, you started the party on the Qubes side! I also only did the ones I'd done, because our trainers at FPF used images already in the Qubes docs, for our journalist training programs... and, well, they were much more difficult to understand in that context. 

A party's a party, and you're the DJ of this one. Dance on!

**Comment by hackerncoder** [comment]
Hey @ninavizz what program do you use to make images like that.

**Comment by hackerncoder** [comment]
Hey @ninavizz what program do you use to make images like that?

**Comment by ninavizz** [comment]
@hackerncoder Sketch, usually. Sometimes Photoshop. Whichever will bog down my computer less, which is usually Sketch. 

I also use a Mac as my daily-driver, and have been using Adobe tooling on a Mac for +30yrs... so more use it when I do because it's a brainless default, than because I have some affinity for Adobe or luxury laptops. I actually kind of detest Adobe, for all the same reasons most FOSS folks do (and because they have zero interest in prioritizing system performance for users). It's just easier for me to use as few tools as possible—and you can't do advanced pre-press/printing stuff with libre tools. Many designers in FOSS use Figma, which I've tried but find to be problematic for various reasons.

If you have Sketch or Photoshop or other tools that could import my files, I'd be happy to share 'em with ya! :)

BTW, this is the double-Whonix-free update on that Tray menu. Sorry to suck at getting it to you more timely!

![TrayRegions_Again](https://user-images.githubusercontent.com/8262612/113227823-6b464500-9248-11eb-9643-aa6c9c4443fb.png)

**Comment by deeplow** [comment]
> If you have Sketch or Photoshop or other tools that could import my files, I'd be happy to share 'em with ya! :)

@hackerncoder there is also photopea.com which is essentially photoshop but browser-based. 99% of the functionalities one would ever need are present. Plus it opens and exports to photoshop assets (`.psd`) as well as many other formats.

**Comment by hackerncoder** [comment]
@ninavizz I don't know if I have any, of course I can always use photopea for photoshop, but yes if you would share them that would be nice.

**Comment by ninavizz** [comment]
@hackerncoder Figma would probably be the easiest way for me to share stuff, tbh. I curse at it, but moreso dislike the operating silo my bouquet of proprietary software boxes me into. Photoshop I only use these days when needing to do elaborate doctoring of screenshots. For the multi-layered stuff, Figma is the way to go. I'll put something together over the weekend.

@deeplow That is a RAD tip, thank you! The fancy preepress stuff it won't have, but tbh 99.999% of folks just never need that part of things (and for digital, nobody does, lol).

**Comment by ninavizz** [comment]
Ok! @hackerncoder @deeplow I have the stuff up in Figma, now, but have to figure-out how to get my sweet proprietary fonts up there. Nope, I am not a fan of open source fonts. If that won't work, I'll switch to using some open source fonts. Yes, I like to ensure type designers get paid. It's a very specialized craft, and one it takes years for people to become good at. Anyone can make a font, but few make them well. And yes, I am a type snob, lol.

**Comment by hackerncoder** [comment]
Wow, already a month. As always time flies. I can see you have a lot on your plate, so no rush @ninavizz.

**Comment by ninavizz** [comment]
Aack! Yes, the appmenu user research proved to be quite the time-bandit. This, and the community-created Simple User Guide, are both on my radar, though! Catching up with another patient client this week, tho. :D

**Comment by ben-grande** [comment]
This solution proposed to the system tray was added to the docs: https://doc.qubes-os.org/en/latest/introduction/getting-started.html#user-interface

I am not closing because it may be worth it to do such thing again, since the icons changed. May also be useful for other parts of the system.
